website/docs/for-react-native/how-to/pull-in-web-bundle.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How to Use a Web App in a Portal
+sidebar_label: Use a Web App in a Portal
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In order to use web applications in your native applications, you'll need to properly setup your project to be able to include a web bundle.
+
+## Setup the Web Asset Directory
+
+Follow the instructions for both Android and iOS in your native applications. You should make the start directories the same between both platforms to avoid issues rendering a Portal.
```

Comment:
Now that this document is separate, we should probably link to the relevant pages for iOS and Android.